spec/fixtures/dos/healthcheck/composite_health_fail.json

```diff
@@ -0,0 +1,23 @@
+{
```

Comment:
I'd appreciate feedback on whether this mock failure file looks reasonable.

Reply:
Thanks @amirbey, I'll change the case for the downstream systems' status.  Any thoughts on what the `comments` field should be for the downstream systems?